util/util.go

```diff
@@ -0,0 +1,53 @@
+package util
+
+import (
+	"strings"
+)
+
+// Logger is used by Session to write logs
+type Logger interface {
```

Comment:
This was initially inside wsmux, but it feels more intuitive to keep this in a separate package.